game/script-jokes.rpy

```diff
@@ -631,3 +659,74 @@ label joke_con_crete:
     extend 1nchgn "-crete!"
 
     return
+
+label joke_footless_snakes:
+    n 1ulraj "Hey,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " have you ever wondered how to measure snakes?"
+    n 1uwdaj "Like they are so long!"
+    n 1fsqcs "What measurement could be possibly the best?"
+    n 1fcsss "Well,{w=0.2} my guess would be in inches or centimetre...{w=0.5}"
+    n 1nsrss "Because they...{w=1}{nw}"
+    extend 1nchgn " have no {i}feet{/i}!"
+
+    return
+
+label joke_bigger_ball:
+    n 1ullbo "You know..."
+    n 1nslss "When I was playing sports lately..."
+    n 1uwdaj "I wondered why the ball keeps getting bigger!{w=0.5}"
+    n 1flrfl "Then it{w=1}{nw}" 
+    extend 1fsqcs " {i}hit{/i} me!"
+
+    return
+
+label joke_meeting_walls:
+    n 1fsqfl "...What did the wall say to the other wall?"
+    n 1fcspo "..."
+    n 1fslbo "We'll meet at the{w=1}{nw}"
+    extend 1fsqbo " {i}corner{/i}."
+
+    return
+
+label joke_hour_feeling:
+    n 1fcsaj "So!{w=1}{nw}"
+    extend 1fsqsm " What did the clock ask the watch,{w=0.2} [player]?"
+    n 1fsldv "..."
+    n 1fchgn "{i}Hour{/i}{w=1} you doing?"
+
+    return
+
+label joke_spotless_tigers:
+    n 1ulraj "So,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " did you ever ask yourself why tigers have stripes?"
+    n 1fcsss "It is not a fashion choice,{w=0.2} that's for sure!"
+    n 1fsqsm "It is because they don't want to be{w=1}{nw}"
```

Comment:
```suggestion
    n 1fcsss "It's not a fashion choice,{w=0.2} that's for sure..."
    n 1fsqsm "...It's because they don't want to be{w=0.5}{nw}"
```

A good attempt here! The main concern is the use of formal English - Nat is a pretty casual speaker, so *it is* comes across as a little too formal/stiff for Nat here.